portal/models/organization.py

```diff
@@ -726,16 +734,16 @@ def populate_tree(self):
         def add_descendents(node):
             partOf_id = node.id
             for org in Organization.query.filter(and_(
-                Organization.id != 0,  # none of the above doesn't apply
-                Organization.partOf_id == partOf_id)):
+                    Organization.id != 0,  # none of the above doesn't apply
+                    Organization.partOf_id == partOf_id)):
```

Comment:
Can you move `):` to a new line, with the same indentation as the `for`, to make it obvious it's not part of the `for` block?
ie:
```
for org in Organization.query.filter(and_(
    ...
):
```